samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -280,25 +280,43 @@ class DailyCallManager {
 
     switch (track.state) {
       case 'playable':
-        videoContainer.style.display = '';
         videoEl.style.display = '';
         break;
       case 'off':
       case 'interrupted':
       case 'blocked':
         videoEl.style.display = 'none'; // Hide video but keep container
         break;
-      // Handle other specific states as needed
+      default:
+        // Here we handle all other states the same as we handle 'playable'.
+        // In your code, you may choose to handle them differently.
+        videoEl.style.display = '';
```

Comment:
Without a proper loading UI, this shows a black screen. I'm more concerned with a good framework than a perfect UI, so I think this is preferred.

Reply:
agree!